### DISCUSSION TITLE
Best way to change the orientation of the camera on the AI Deck to make it downward facing?

### DISCUSSION THREAD

**Discussion by shreeram-murali** [bitcraze/discussions#107] [Q&A] [answered]
Hi everyone! 

I'm working on a project involving landing and I was wondering what's the best/easiest way we could make the camera on the AI deck downward facing. While we could detach (as [documented here](https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/guides/camera-replace/)) the current bracket and make a new one, I was wondering if you have any ideas on a better way to do it. 

Thanks.

**Comment by tobbeanton** [comment, marked as answer]
I would say that is the way to go. You don't need a new bracket (but might be good for protecting it) you can just tape it to the bottom side. The camera already has a sticky pad on it that is not used in the normal mount.

**Reply by shreeram-murali** [reply]
Thanks! This seems like the way to go, although I’m a bit concerned about the camera’s cable being over-stretched.
